GNUmakefile

```
@@ -431,6 +431,10 @@ missing: ## Check for packages not being tested
 cl: ## Create a new Changelog entry
 	@go run -modfile tools/go.mod tools/cl-entry/main.go
 
+.PHONY: cltui
+cltui: ## Create a new Changelog entry
```

Just so they have different help text:

```suggestion
cltui: ## Create a new Changelog entry interactively
```

Presumably we're not replacing yet but that should probably be the long-term goal so we don't have extra tools littered around :grinning: